packages/cascara/src/modules/ModuleError/ModuleError.js

```diff
@@ -16,9 +28,14 @@ const ModuleError = ({ moduleName, moduleOptions }) => {
   console.error(message);
 
   return (
-    <div className={styles.Error} data-testid={'module-error'}>
+    <Role
+      {...rest}
+      as={as}
+      className={cx(className, 'Error')}
+      data-testid={'module-error'}
+    >
       {message}
-    </div>
+    </Role>
```

Comment:
We do not need to use the Role component here as this ModuleError component is not exported from Cascara. It is a private component.